docs: update vim docs to no longer point people to coc-metals (#3819)
scalameta/metals: docs/editors/vim.md

title: Vim
---

![nvim-metals demo](https://i.imgur.com/BglIFli.gif)

While Metals works with most LSP clients for [Vim](https://www.vim.org/) and
[Neovim](https://neovim.io/), we recommend using the dedicated Neovim plugin to
get the best Metals support. Metals has many specific commands and LSP
extensions that won't be available when not using the extension.

```scala mdoc:requirements

```

## nvim-metals

[`nvim-metals`](https://github.com/scalameta/nvim-metals) Is the the dedicated
Metals extension for the [built-in LSP
support](https://neovim.io/doc/user/lsp.html) in Neovim.

To get started with installation please see the `nvim-metals`
[prerequisites](https://github.com/scalameta/nvim-metals#prerequisites) and
[installation steps](https://github.com/scalameta/nvim-metals#installation).

```scala mdoc:releases

```

Keep in mind that by default Neovim doesn't have default mappings for the
functionality you'll want like, hovers, goto definition, method signatures, etc.

> Youc an find a full example configuration of these in the [example

Reviewer: ```suggestion
You can find a full example configuration of these in the [example
```

Author: Thanks, fixed!
